tests/general/getentropy.c

```diff
@@ -0,0 +1,17 @@
+#include <assert.h>
+#include <unistd.h>
+
+int main() {
+  char buf[256] = {0};
+  int ret = getentropy(buf, 256);
+  assert(ret == 0);
+
+  int sum = 0;
+  for (int i = 0; i < 256; i++) {
+    sum += buf[i];
```

Comment:
Maybe use unsigned types and `^=` instead of `+=` to check for constant output (instead of specifically `0` output).